system/Filters/Filters.php

```diff
@@ -104,13 +112,52 @@ class Filters
 	 * @param \Config\Filters   $config
 	 * @param RequestInterface  $request
 	 * @param ResponseInterface $response
+	 * @param \Config\Modules	$moduleConfig
 	 */
-	public function __construct($config, RequestInterface $request, ResponseInterface $response)
+	public function __construct($config, RequestInterface $request, ResponseInterface $response,$moduleConfig = null)
```

Comment:
need \Config\Modules type hint

Reply:
Hi ther, I am little confused how todo this, copied from RouterCollection so that I try and follow the standard and it looks like this , how do I do that Config type hint ?
![prent](https://user-images.githubusercontent.com/33496470/90851148-1e29f880-e374-11ea-9bf0-2c13c47a964f.png)

Reply:
K I think I have booked a solution in that will work, I looked at the FileLocator in RouteCollection and copied the same concept from it for Modules.